src/utils/redis-search-stack/build-search-query.js

```diff
@@ -79,6 +81,28 @@ const searchQueryBuilder = {
 
     return expression(field, tokensMatch(params, true));
   },
+  any: (prop, field, expr) => {
+    const propName = normalizePropName(prop);
+
+    if (prop.endsWith('_tag')) {
```

Comment:
It's better ti remove `_tag`-postfix dependency

The `eq` function from SearchQueryBuilder handle exact match using `{ tag }` expression